src/bullsAndCows.js

```diff
@@ -26,7 +26,32 @@
  * @return {object} - like {bulls: 0, cows: 0}
  */
 function bullsAndCows(generatedNumber, enteredNumber) {
-  // write code here
+  if (typeof enteredNumber === 'undefined'
+    || enteredNumber.length !== 4
+    || enteredNumber
+      .split('')
+      // eslint-disable-next-line max-len
+      .filter((item, index, array) => array.lastIndexOf(item) === array.indexOf(item)).length !== enteredNumber.length) {
```

Comment:
That's way to complex and hard to read. You can check if there are unique digits only in a simpler way.